### ISSUE TITLE
Add Support for HomeyScript

### ISSUE THREAD

**Issue by tboudri** [jeroenbos22/homey.countdown#13]
At this time I am using CountDown. But since it does not have support for HomeyScript i am looking for an alternative

Can you add HomeyScript support to start/stop and get/set timer values?

Many Thanks

**Comment by Milanzor** [comment]
I'll have a look into it, I've not used HomeyScript myself but perhaps it's easy to implement. Will post back when I know more

**Comment by selzilla** [comment]
Is the app compatible with homey 2.0 rc11? I can not create a countdown anywhere.

**Comment by jeroenbos22** [comment]
@selzilla Yes app is compatible, if you use the stable version you can add timers using the action card. If you have the beta version you can add them through app settings.

**Comment by jeroenbos22** [comment]
@tboudri Do you know how to make the app work with homeyscript? We're always open for a cool pull request ;)

**Comment by Schamper** [comment]
FWIW, you can use the existing API endpoints in HomeyScript:
```javascript
let app = await Homey.apps.getApp({id: 'homey.countdown'});
await app.apiPost('/update_timer/', {timername: 'name', seconds: 1337});
```
